pyispyb/core/schemas/datacollections.py

```diff
@@ -155,6 +167,8 @@ class DataCollection(DataCollectionBase):
 
     DataCollectionGroup: DataCollectionGroup
     GridInfo: Optional[list[GridInfo]]
+    SSXDataCollection: Optional[sqlalchemy_to_pydantic(models.SSXDataCollection)]
```

Comment:
This is more complicated because its nested and not generated at run time. The easy option is to manually define the Schema in this case. Will have to think about runtime possibilities...